torch/fx/passes/utils/matcher_utils.py

```diff
@@ -228,7 +228,25 @@ def _match_args(args1: Union[List, Tuple], args2: Union[List, Tuple]) -> bool:
 
             return True
 
-        match_found = match_found and _match_args(pn.args, gn.args)
+        pn_args, gn_args = [], []
```

Comment:
just for my understanding, what happens if we have:
schema: op(a, b=1, c=2) and 
pattern: op(a, c=2), subgraph: op(a, b=2)?

do we just fill in the default values here for node.args and ignore the kwargs? but kwargs will be matched in L252-L259?

Reply:
Yeah you're right -- in those cases these wouldn't get matched. I was just thinking in the case of aten ir, where there are no kwargs. Let me also add the case for kwargs too.

Reply:
updated!